solutions/security/ai/attack-discovery.md

```diff
@@ -19,4 +17,105 @@ $$$attack-discovery-generate-discoveries$$$
 
 $$$attack-discovery-what-info$$$
 
-$$$attack-discovery-workflows$$$
\ No newline at end of file
+$$$attack-discovery-workflows$$$
+
+# Attack Discovery [attack-discovery]
+
+::::{warning}
+This feature is in technical preview. It may change in the future, and you should exercise caution when using it in production environments. Elastic will work to fix any issues, but features in technical preview are not subject to the support SLA of GA features.
+::::
+
+
+Attack Discovery leverages large language models (LLMs) to analyze alerts in your environment and identify threats. Each "discovery" represents a potential attack and describes relationships among multiple alerts to tell you which users and hosts are involved, how alerts correspond to the MITRE ATT&CK matrix, and which threat actor might be responsible. This can help make the most of each security analyst’s time, fight alert fatigue, and reduce your mean time to respond.
+
+For a demo, refer to the following video.
+
+<!-- ::::{admonition}
```

Comment:
The current syntax guide doesn't contain guidance for videos. Florent seemed to have figured out a workaround (see [this Slack thread](https://elastic.slack.com/archives/C07APH4RCDT/p1738856683024319)) that we might be able to use if needed.